docs/Shell-Completion.md

```diff
@@ -46,7 +46,7 @@ then
 fi
 ```
 
-This must be done before `compinit` is called. Note that if you are using Oh My Zsh, it will call `compinit` for you, so this must be done before you call `oh-my-zsh.sh`. This may be done by appending the following line to your `~/.zprofile` after Homebrew's initialization, instead of modifying your `~/.zshrc` as above:
+This must be done before `compinit` is called. Note that if you are using Oh My Zsh, it will call `compinit` for you when sourced. In this case, instead of the above, add the following line to your `~/.zshrc`, before you call `oh-my-zsh.sh`:
```

Comment:
How about this (assuming it is correct)?

```suggestion
This must be done before `compinit` is called. Note that if you are using Oh My Zsh, it will call `compinit` for you when you source `oh-my-zsh.sh`. In this case, instead of the above, add the following line to your `~/.zshrc`, before you source `oh-my-zsh.sh`:
```

Reply:
Sure! Didn't want to be repetitive but this is correct 👍